controllers/result.go

```diff
@@ -204,8 +204,8 @@ func getLatestSubmissionKey(ctx context.Context, resultKey, taskKey *datastore.K
 	return keys[0], nil
 }
 
-func computeFinalScore(ctx context.Context, challenge model.Challenge, result model.Result, resultKey *datastore.Key) {
-	if err := logic.Resulter(challenge.Resulter).Call(ctx, resultKey); err != nil {
+func computeFinalScore(ctx context.Context, challenge model.Challenge, result model.KeyedResult) {
```

Comment:
The ordering of params is backwards. Would be nice to have the same order as for `logic.Resulter`